pycbc/waveform/waveform.py

```diff
@@ -42,6 +42,8 @@
                       spa_tmplt_precondition, spa_amplitude_factor, \
                       spa_length_in_time
 
+import gwsignal
```

Comment:
@bhooshan-gadre Maybe whatever this package is importing should just export the standard pycbc waveform interface? We have a plugin architecture so packages can advertise support for the pycbc waveform interfaces. 

https://pycbc.org/pycbc/latest/html/waveform_plugin.html

Reply:
@ahnitz Soon, even the lalsuite waveform infrastructure will be modified. So apart from the new waveform, support for lalsuite-based waveforms may break in upcoming releases. 

I will try to use pycbc waveform interface as well and then we can decide.

Reply:
We also need special lalsuite (with hash 391604dba2) for this.